client/repodlgs_common.cpp

```diff
@@ -150,6 +150,63 @@ void get_economy_report_units_data(struct unit_entry *entries,
   unit_type_iterate_end;
 }
 
+/**
+   Returns an array of units data. Number of units in
+   the array is added to num_entries_used.
+ */
+void get_units_view_data(struct unit_view_entry *entries,
+                         int *num_entries_used)
+{
+  int count, gold_cost, food_cost, shield_cost;
+
+  *num_entries_used = 0;
```

Comment:
This doesn't follow the documentation ("Number of units in the array is added to num_entries_used.")